src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -322,7 +387,13 @@ def get_counter(self, sub_id, table_name):
             counter_value = 0
             for lag_member in self.lag_name_if_name_map[self.oid_lag_name_map[oid]]:
                 counter_value += self._get_counter(mibs.get_index_from_str(lag_member), table_name)
-
+            sai_lag_id = self.lag_sai_map[self.oid_lag_name_map[oid]]
+            sai_lag_rif_id = self.port_rif_map[sai_lag_id]
+            if sai_lag_rif_id in self.rif_port_map:
+                table_name = getattr(table_name, 'name', table_name)
```

Comment:
why we need getattr here?

Reply:
Stepan said the purpose is to return the entry in table_name corresponding to 'name', but if there is no 'name' attribute, return table_name.  There are other ways of doing it, but this is the most concise.

Reply:
It looks strange, could you add some comment here? I suppose table_name should be a str which has no 'name' attribute, could you please explain the situation that table_name would have a 'name' attribute?